### DISCUSSION TITLE
taro 分包时app.wxss文件报分包产生的common.wxss文件not find 但是路径时正确的

### DISCUSSION THREAD

**Discussion by zhanshihua** [NervJS/taro#14496] [Q & A]
<img width="1119" alt="image" src="https://github.com/NervJS/taro/assets/67133914/6fd6645d-b070-4cf7-8fa4-0eecb54d8632">
<img width="1155" alt="image" src="https://github.com/NervJS/taro/assets/67133914/b9ecc5b8-e8e0-4efe-a656-791e26b02ffb">

**Comment by XiaoDaoGitHub** [comment]
我也碰到了这个问题，解决了吗
